Stardrop/ViewModels/DownloadPanelViewModel.cs

```diff
@@ -0,0 +1,200 @@
+﻿using Avalonia.Controls;
+using DynamicData;
+using DynamicData.Aggregation;
+using DynamicData.Alias;
+using DynamicData.Binding;
+using ReactiveUI;
+using Stardrop.Models.Data;
+using Stardrop.Utilities.External;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Stardrop.ViewModels
+{
+    public class DownloadPanelViewModel : ViewModelBase
+    {
+        public string NoDownloadsLabel { get; init; } = Program.translation.Get("ui.downloads_panel.no_downloads_label");
+
+        private ObservableCollection<ModDownloadViewModel> _downloads = new();
+        public ObservableCollection<ModDownloadViewModel> Downloads { get => _downloads; set => this.RaiseAndSetIfChanged(ref _downloads, value); }        
```

Comment:
One thing to note: we store `Downloads` in a plain (observable) list, and we use the download's `Uri` to find it in the list. That has two implications:
1) We can't download things from the same Uri more than one at a time (which is mostly fine because Nexus download links have user IDs and expiry times in them, so it's very rare for them to be the same)
2) If the user has a *lot* of downloads at once, searching the list could get pretty slow. In practice, this seems unlikely, so I decided to go with the simpler implementation.